extensions/adobe/experience/offer-management/domain-model.schema.json

```diff
@@ -0,0 +1,58 @@
+{
+  "meta:license": [
+    "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "$id":
+    "https://ns.adobe.com/experience/offer-management/domain-model",
+  "meta:extensible": "true",
+  "type": "object",
+  "title": "Domain Model",
```

Comment:
domains means too many different things in XDM.  Can we pick a different term here?

Reply:
The instances of this schema act as a filter for the underlying schemas. This allows the solution architect to "publish" some properties for use in rules, filters, queries and constraints while keeping others private. This is effectively a bridge between a raw data schema and the set of properties displayed to the end user in UI.

Reply:
Now it's `schema-projection` not `domain-model`